### ISSUE TITLE
Error reading metadata when using Image.getexif().tobytes()

### ISSUE THREAD

**Issue by lyaoworkaccount** [python-pillow/Pillow#7185]
### What did you do?

Using Pillow to read the exif metadata of a jpg image

### What did you expect to happen?

Expected the metadata to converted into bytes

### What actually happened?

Issue in the tobytes() function reading the metadata 

```
python3.9/site-packages/PIL/TiffImagePlugin.py", line 895, in tobytes
    "<table: %d bytes>" % len(data) if len(data) >= 16 else str(values)
TypeError: object of type 'IFDRational' has no len()
```

### What are your OS, Python and Pillow versions?

* OS: Ubuntu 22.04.2  
* Python: 3.9.9
* Pillow: 9.4.0

```python
from io import BytesIO
from pathlib import Path
from PIL import Image

with Path(__file__).with_name("sample.jpg").open("rb") as fp:
    image = Image.open(BytesIO(fp.read())) 
    image.getexif().tobytes()
```

a whited out 1x1 pixel image with the metadata that has this problem is attached below

![sample](https://github.com/python-pillow/Pillow/assets/134630375/20c12c6f-70b9-4293-9efb-d169887df801)

**Comment by radarhere** [comment]
Hi. This has already been resolved by #6890. If you upgrade to Pillow 9.5.0, your problem should be fixed.

**Comment by lyaoworkaccount** [comment]
Thanks!
